packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -21,24 +23,21 @@ const { generateTemplate } = storybookTemplate('sd-quickfact');
 export default {
   title: 'Components/sd-quickfact',
```

Comment:
Could you show the default in your intro, pls? 

![Uploading image.png…]()


Expandable is an optional feature.

Reply:
When done -> Approved from design side. THX!

Reply:
Sorry, missed this one. 
Updated now.